data/mods/Aftershock/player/mutations.json

```diff
@@ -1,15 +1,4 @@
 [
-  {
-    "type": "mutation",
-    "id": "AFS_PROF_SKIIER",
```

Comment:
The id can change Alpine descendant and the description can reference being descended from various potential people's of the earth from Aleutian, Ugric and Nepalese who actually have epigenetic markers that lead to slight body changes that allow for easier survival in extreme winter temperatures or high altitudes.  You don't have to and I might do it myself and then allow it as a 1 point starting trait? Or we could see about mainlining it? I'm also open to suggestions on this one.

Reply:
Eh personally I wouldnt bring racial traits into the mod, because its a sensitive topic and because its still not sci-fi.

I think it could be worthwhile to turn it into a "dermal insulation bionic" and give it to something like an industrial wage slave type profession.

Reply:
I actually refluffed the skiier into a woken from cryosleep survivor, inspiration suddenly hit me, and it was easy to do.